frontend/package.json

```diff
@@ -108,10 +108,12 @@
     "serve": "next start",
     "test": "playwright test",
     "e2e": "playwright test",
+    "e2e:ci": "playwright test --config playwright-ci.config.ts",
     "e2e:chrome": "playwright test --project chromium",
     "e2e:chrome:headed": "playwright test --headed --project chromium",
-    "e2e:ff": "playwright test --project firefox",
-    "e2e:ie": "playwright test --project edge",
+    "e2e:browser:chrome": "playwright test --workers 1 --project chromium",
```

Comment:
Feels like we could remove these three scripts, because we no longer have e2e-all?

Thank you!

Reply:
ah sorry I meant revert it back to:

```
"e2e:ff": "playwright test --project firefox",
    "e2e:ie": "playwright test --project edge",
```

since I don't think we need `--workers 1` in these scripts?